enhancements/update/automatic-updates.md

```diff
@@ -0,0 +1,166 @@
+Allowing automatic updates would make it more likely that upgrade failures happen when there is no admistrator actively watching to notice and recover from the failure.
+This is mitigated by:
+
+* Alerting.
+    The cluster should fire alerts (FIXME: which ones?) when an upgrade gets into trouble.
+    Adminstrators can configure the cluster to push those alerts out to the on-call admistrator to recover the cluster.
+* Stability testing.
+    We are continually refining our CI suite and processing Telemetry from live clusters in order to assess the stability of each upgrade.
+    We will not place upgrades in production channels unless they have proven themselves stable in earlier testing, and we will remove upgrades from production channels if a live cluster trips over a corner case that we do not yet cover in pretesting.
+
+There are also potential future mitigations:
+
+* Phased releases, where Cincinnati spreads an update suggestion out over a configurable time window.
+    With hard cutovers and automatic updates, many clusters in a given channel could attempt a new update simultaneously.
+    If that update proves unstable, many of those upgrades would already be in progress by the time the first Telemetry comes back with failure messages.
+    A phased rollout would limit the number of simultaneously updating clusters to give Telemetry time to come back so we could stop recommending upgrades that proved unstable on live clusters not yet covered in pretesting.
+
+There is also a security risk where a compromised upstream Cincinnati could recommend cluster updates that were not in the cluster's best interest (e.g. 4.2.4 -> 4.1.0).
+This is mitigated by:
+
+* [The configurable `upstream` property][api-upstream].
+    Administrators who do not trust the default `upstream` to be sufficiently reliable may point their clusters at an upstream that they control which they can secure to their satisfaction.
+    The drawback to this approach would be that now they need tooling to approve graph updates as Red Hat makes changes, which will introduce some delays to security-fix rollouts.
+    But that approval would scale per-custom-Cincinnati instead of scaling per-cluster.
+
+There are also potential future mitigations:
+
+* The cluster-version operator could be taught to limit automatic upgrades to those which appear in the [signed metadata][cluster-version-operator-release-metadata] of either the source or target release.
+    The drawback to this approach is that upgrades would be limited to those expected when the releases were created and signed.
+
+## Design Details
+
+### Test Plan
+
+In addition to unit tests, this feature will be tested in a fleet of canary clusters which will run with automatic updates enabled.
+Having the canaries successfully and automatically update in to and out of a candidate release will be part of release and update stabilization criteria.
```

Comment:
Can you maybe expand a bit more the "automatically update ... out of a candidate release" bit?
I think it's important to spell out the plan to ensure that the auto-update logic in the candidate release are not broken; in my experience so far that was always a blind spot.

Reply:
> Can you maybe expand a bit more the "automatically update ... out of a candidate release" bit?

Done in 9b61a96 -> 568a1bd.